src/main/java/at/hannibal2/skyhanni/features/nether/reputationhelper/CrimsonIsleReputationHelper.kt

```diff
@@ -42,6 +43,11 @@ class CrimsonIsleReputationHelper(skyHanniMod: SkyHanniMod) {
     private var display = emptyList<List<Any>>()
     private var dirty = true
 
+    val tabListQuestPattern by RepoPattern.pattern(
+        "crimson.reputation.tablist",
+        " §r§d.*"
```

Comment:
```suggestion
        " §r§[cdea].*"
```
This should work.
* `c` - Barbarian Not Accepted
* `d` - Mage Not Accepted
* `e` - Accepted
* `a` - Completed